bin/conjur-cli.rb

```diff
@@ -75,16 +75,15 @@ def test_select
 command :policy do |cgrp|
   cgrp.desc "Load MAML policy from file(s)"
   cgrp.arg :account
-  cgrp.arg_name "filename"
-  cgrp.arg :file_name, :multiple
+  cgrp.arg :filename, :multiple
   cgrp.command :load do |c|
     c.action do |global_options,options,args|
       account, *file_names = args
       connect
 
-      file_names.each do |file_name|
-        exec "rake policy:load[#{account},#{file_name}]"
-      end
+      file_names.map { |file_name|
+        system "rake policy:load[#{account},#{file_name}]"
+      }.all?
```

Comment:
You still need to use `exit_now!` (or just `fail`) -- it appears that GLI doesn't look at the return value of an action.